src/FsYacc/AssemblyInfo.fs

```diff
@@ -5,13 +5,13 @@ open System.Reflection
 [<assembly: AssemblyTitleAttribute("FsYacc")>]
 [<assembly: AssemblyProductAttribute("FsLexYacc")>]
 [<assembly: AssemblyDescriptionAttribute("FsLex/FsYacc lexer/parser generation tools")>]
-[<assembly: AssemblyVersionAttribute("9.0.3")>]
-[<assembly: AssemblyFileVersionAttribute("9.0.3")>]
+[<assembly: AssemblyVersionAttribute("9.1.0")>]
```

Comment:
These files could be ignored by Git. The commits would have been much less cluttered.